### DISCUSSION TITLE
I bought the domain renewal it didn't renew !!!

### DISCUSSION THREAD

**Discussion by PedroTavolaro** [vercel/community#2289] [Help] [answered]
### Summary

I bought the pedrotavolaro.com domain renewal but it didn't renew, how can I solve this problem? How can I contact support?

### Additional information

I bought the pedrotavolaro.com domain renewal but it didn't renew, how can I solve this problem? How can I contact support?

**Comment by amyegan** [comment, marked as answer]
Hi @PedroTavolaro! It looks like someone already helped you with domain renewal in your support case. If you need more help in the future, you can ask here in Vercel Community Discussions or [submit a case](https://vercel.com/help#issues) to the Customer Success Team
